src/app/api/v1/applications/[appSlug]/users/login/route.ts

```diff
@@ -0,0 +1,24 @@
+import { NextRequest, NextResponse } from "next/server";
+import { StatusCodes } from "http-status-codes";
+import { sendVerificationEmailCode } from "@/lib/emails/auth/sendVerificationEmailCode";
+
+export async function POST(req: NextRequest) {
+  const body = await req.json();
+  const { email } = body;
+
+  const otpCodeSent = await sendVerificationEmailCode({
```

Comment:
here you will have a conflict cause i've splitted this function into 2 seperates one. 
1 for gerneating OTP, 2 for sending email (the generic one)